app/views/components/fields/removable_elements_table.scala.html

```diff
@@ -23,12 +23,14 @@
 
 <div class="field-group mb-2">
     <table id="@{id}">
-      @title.map { heading =>
-          <thead>
-              <tr>
-                <th id="@{id}-heading" colspan="2">@heading</th>
-              </tr>
-          </thead>
+      @if(elements.nonEmpty){
```

Comment:
So if the elements are empty we dont want to show the table title?

Would it not be better to have this `if` block around the entire `table/div`?

Reply:
Maybe around the table.  Getting rid of the div might affect the layout/spacing?   Also, there might be tests relying on finding a table by id.   

Thought this was the least invasive change

Reply:
I would presume the tests wouldn't look for the table by ID if it shouldn't be there though?

Reply:
I wouldn't worry about spacing, the margin/padding is on the wrong element if it is required even when the table isn't there